text/getting_started.tex

```diff
@@ -72,13 +72,14 @@ \section{Getting the RTOS source code}
 
 If, like me, your preferred development environment is a lightly modernised version of a 1970s minicomputer, you can still use the dev container.
 The container image is \file{ghcr.io/cheriot-platform/devcontainer:latest}.
-You can run this directly (from the directory where you checked out the RTOS repository):
+You can run this directly from the directory where you checked out the RTOS repository:
 
 \begin{console}
-$ docker run --rm -it \\
-	--mount source=$(pwd),target=/home/cheriot/cheriot-rtos,type=bind \\
+$ docker run --rm -it --mount \\
+	source=$(pwd),target=/home/cheriot/cheriot-rtos,type=bind \\
```

Comment:
```suggestion
$ docker run --rm -it \\
	--mount \\
 	source=$(pwd),target=/home/cheriot/cheriot-rtos,type=bind \\
```

This makes it a bit more clear that the next bit is the argument to the mount flag.